bin/install

```diff
@@ -82,6 +89,23 @@ get_java() {
 		--retry 3 --retry-delay 3 \
 		-b oraclelicense=a \
 		-o $destdir/$variant $url
+    
+	if [ "$signature" != "" ] && [ "$sigkey" != "" ]; then
```

Comment:
Do we want to make signature verification optional?

Reply:
@SISheogorath only for the other (older) distributions that do not set the vars

Reply:
I see.